<launch>
<node pkg="forklift_server" type="PBVS_server_4WDD.py" name="PBVS_server" output="screen">
    
    <!--Subscriber Topic setting/-->
    <param name="odom" value="/wheel_odom" /><!--里程計 Topic（map -> base_link, 是tf相對關係, 非輪式里程計）/-->
    <param name="tag_detections_topic" value="/tag_detections" /><!--車體相機對位AprilTag Topic/-->

    <!--bodycamera parking setting-->
    <param name="bodycamera_tag_offset_x" type="double" value = "0.0" /><!--對位目標點與tag的左右偏移量/-->
    <param name="bodycamera_ChangingDirection_threshold" type="double" value = "0.05" />  <!--正對Tag(atan2(y, x) <= threshold)/-->
    <param name="bodycamera_parking_stop" type="double" value = "0.12" />  <!--車體相機對位停止距離/--> <!-- 0.16 -->
    <param name="bodycamera_Changingtheta_threshold" type="double" value = "0.06" />  <!--平行Tag(theta <= threshold)/-->
    <param name="bodycamera_decide_distance" type="double" value = "0.1" />  <!--將相機平行Tag之後，左右y距離誤差幾公尺內可以將牙叉插進去，如果不行的話就後退重新對位/-->
    <param name="bodycamera_back_distance" type="double" value = "1.40" />  <!--重新對位時後退的距離/-->
</node>
</launch>
